blenderproc/python/writer/BopWriterUtility.py

```diff
@@ -197,22 +199,27 @@ def _get_frame_gt(dataset_objects: List[bpy.types.Mesh], unit_scaling: float, ig
 
         frame_gt = []
         for obj in dataset_objects:
-
-            H_m2w = Matrix(WriterUtility.get_common_attribute(obj, 'matrix_world'))
+            if type(obj) == Link:
```

Comment:
```suggestion
            if isinstance(obj, Link):
```

That would also work with derived classes.